### PR TITLE
Accessibility Forgot password #2221

### PR DESCRIPTION
**Pull request by idzhersh**
### Before:
https://docs.google.com/spreadsheets/d/116m-cqLYVSYjLp4Zeosm_NTDoeb7XCdIFd70gTpPJe8/edit#gid=820121178
313 - Focus land on the bottom of the page after pressing on 'Close graphic link button'
314 - Close 'Forgot password' popup button is not focusable via tab navigation and is recognized as 'graphic link close button' for SR.
315 - 'Remember your password' is plain text instead of heading.

### After
313 - Focus land on the Sign In or Sign up link after pressing on 'Close graphic link button'
314 - Close 'Forgot passworwod' popup button is focusable via tab navigation and is recognized as "close forgot password button" for SR
315 - 'Remember your password' is h3 tag

### CODE REVIEW

**Review by yuriiLisunov — APPROVED** [review]

**Review by KirillAbramo — APPROVED** [review]

File: src/app/component/layout/components/header/header.component.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { headerIcons } from './../../../../image-pathes/header-icons';
-import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Component, OnInit, OnDestroy, ViewChild, ElementRef } from '@angular/core';
 import { NavigationStart, Router } from '@angular/router';
-import { MatDialog } from '@angular/material';
-import { filter, takeUntil } from 'rxjs/operators';
+import { MatDialog} from '@angular/material';
+import { filter, take, takeUntil } from 'rxjs/operators';
```
Comment (JimHolman1199) [inline src/app/component/layout/components/header/header.component.ts:5]: Remove this unused import of 'take'.

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "@ngrx/effects": "^9.2.0",
     "@ngx-translate/core": "^11.0.1",
     "@ngx-translate/http-loader": "^4.0.0",
-    "@stomp/rx-stomp": "^1.0.1",
+    "@stomp/ng2-stompjs": "^8.0.0",
```
Comment (JimHolman1199) [inline package.json:34]: I do not fully understand why these changes will take place?

File: src/app/component/auth/components/auth-modal/auth-modal.component.ts [hunk 3]
```diff
@@ -12,6 +12,7 @@ export class AuthModalComponent implements OnInit {
   public authImages = authImages;
   public authPage: string;
 
+
```
Comment (JimHolman1199) [inline src/app/component/auth/components/auth-modal/auth-modal.component.ts:15]: remove space

File: src/app/component/auth/components/auth-modal/auth-modal.component.ts [hunk 4]
```diff
@@ -30,9 +31,11 @@ export class AuthModalComponent implements OnInit {
   }
 
   public closeWindow(): void {
-    this.matDialogRef.close();
+    this.matDialogRef.close( );
   }
 
+
```
Comment (JimHolman1199) [inline src/app/component/auth/components/auth-modal/auth-modal.component.ts:37]: format correctly

File: src/app/component/layout/components/header/header.component.ts [hunk 5]
```diff
@@ -22,7 +22,7 @@ import { Subject } from 'rxjs';
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss']
 })
-export class HeaderComponent implements OnInit, OnDestroy {
+export class HeaderComponent implements OnInit,  OnDestroy {
```
Comment (JimHolman1199) [inline src/app/component/layout/components/header/header.component.ts:25]: why do you need space before OnDestroy  ?

File: src/app/component/layout/components/header/header.component.ts [hunk 6]
```diff
@@ -216,4 +231,5 @@ export class HeaderComponent implements OnInit, OnDestroy {
       document.body.classList.add('modal-open') :
       document.body.classList.remove('modal-open');
   }
+
```
Comment (JimHolman1199) [inline src/app/component/layout/components/header/header.component.ts:234]: again )

File: package.json (hunk 2 above)
Comment (idzhersh) [reply package.json:34]: the test failed. maybe this version that works properly with LinuxOS. After replacement, the test did not failed

File: src/app/component/auth/components/auth-modal/auth-modal.component.ts (hunk 3 above)
Comment (idzhersh) [reply src/app/component/auth/components/auth-modal/auth-modal.component.ts]: Done

Comment (idzhersh) [reply src/app/component/auth/components/auth-modal/auth-modal.component.ts]: Done

File: src/app/component/layout/components/header/header.component.ts (hunk 1 above)
Comment (idzhersh) [reply src/app/component/layout/components/header/header.component.ts]: Done

**Review by idzhersh — COMMENTED** [review]
after commits updates

**Review by ZooMingGG — APPROVED** [review]

**Review by mr-pavlenko — APPROVED** [review]
